provider/resources.go

```diff
@@ -4475,6 +4475,18 @@ $ pulumi import aws:networkfirewall/resourcePolicy:ResourcePolicy example arn:aw
 					{Name: "M6id_24XLarge", Value: "m6id.24xlarge"},
 					{Name: "M6id_32XLarge", Value: "m6id.32xlarge"},
 					{Name: "M6id_Metal", Value: "m6id.metal"},
+					{Name: "M7a_Medium", Value: "m7a.medium"},
+					{Name: "M7a_Large", Value: "m7a.large"},
+					{Name: "M7a_XLarge", Value: "m7a.xlarge"},
+					{Name: "M7a_2XLarge", Value: "m7a.2xlarge"},
+					{Name: "M7a_4XLarge", Value: "m7a.4xlarge"},
+					{Name: "M7a_8XLarge", Value: "m7a.8xlarge"},
+					{Name: "M7a_12XLarge", Value: "m7a.12xlarge"},
+					{Name: "M7a_16XLarge", Value: "m7a.16xlarge"},
+					{Name: "M7a_24XLarge", Value: "m7a.24xlarge"},
+					{Name: "M7a_32XLarge", Value: "m7a.32xlarge"},
+					{Name: "M7a_48XLarge", Value: "m7a.48xlarge"},
+					{Name: "M7a_Metal", Value: "m7a.metal"},
```

Comment:
I'm tracking this against https://aws.amazon.com/ec2/instance-types/ for all fields up to `m7a.metal`. On AWS's website, I see `m7a.metal-48xl`. 
```suggestion
					{Name: "M7a_Metal", Value: "m7a.metal-48xl"},
```

Reply:
Thanks @iwahbe. I think I missed it. I will update it and commit.

Reply:
Hi @iwahbe, I just updated it